source/uwp/Renderer/dll/dll.cpp

```diff
@@ -2,15 +2,14 @@
 //    Copyright (C) Microsoft.  All rights reserved.
 //
 #include "pch.h"
```

Comment:
I would suggest we remove dll.cpp.

then in the .def

EXPORTS
DllCanUnloadNow = WINRT_CanUnloadNow                    PRIVATE
DllGetActivationFactory = WINRT_GetActivationFactory    PRIVATE

That's what the cppwinrt template did when I create a cpp/winrt from visual studio with the cppwinrt template